### PR TITLE
Some minor CS fixes

### PR DESCRIPTION
**Pull request by aRn0D**
Related to #92

### CODE REVIEW

**Comment by webmozart** [comment]
Awesome, thanks @aRn0D! :)

File: src/Resource/Iterator/ResourceCollectionIterator.php [hunk 1]
```diff
@@ -165,7 +165,7 @@ public function next()
     public function key()
     {
         if (null === ($key = key($this->resources))) {
-            return null;
+            return;
```
Comment (webmozart) [inline src/Resource/Iterator/ResourceCollectionIterator.php]: Could you undo this change? Since `key()` returns a value, `return null;` should be preferred.

**Comment by aRn0D** [comment]
@webmozart Updated!

**Comment by webmozart** [comment]
Thanks :) Could you apply the StyleCI fixes and squash your commits please?

**Comment by aRn0D** [comment]
@webmozart I will fix it. Squash my commits? I have only one commit :)

**Comment by webmozart** [comment]
Ah yes, you're right :)

**Comment by aRn0D** [comment]
@webmozart Can you restart scrutinizer? Dont know why it fails...

**Comment by webmozart** [comment]
Done

**Comment by webmozart** [comment]
Thanks @aRn0D!

**Comment by aRn0D** [comment]
You're welcome :v:
